core/src/test/scala/org/apache/spark/MapStatusesSerDeserBenchmark.scala

```diff
@@ -85,12 +86,11 @@ object MapStatusesSerDeserBenchmark extends BenchmarkBase {
     }
 
     benchmark.run()
-    // scalastyle:off
-    import org.apache.commons.io.FileUtils
+    // scalastyle:off println
     benchmark.out.println("Compressed Serialized MapStatus sizes: " +
-      FileUtils.byteCountToDisplaySize(serializedMapStatusSizes))
```

Comment:
shall we update the benchmark golden files?

Reply:
I'm doing that.

Reply:
Thank you, and excuse me for being a bit pushy.

Reply:
The corresponding benchmark results are updated